src/openzaak/components/besluiten/admin.py

```diff
@@ -1,22 +1,39 @@
 from django.contrib import admin
 
-from openzaak.utils.admin import AuditTrailAdminMixin, AuditTrailInlineAdminMixin
+from openzaak.utils.admin import (
+    AuditTrailAdminMixin,
+    EditInlineAdminMixin,
+    ListObjectActionsAdminMixin,
+    UUIDAdminMixin,
+    link_to_related_objects,
+)
 
 from .models import Besluit, BesluitInformatieObject
 
 
-class BesluitInformatieObjectInline(AuditTrailInlineAdminMixin, admin.TabularInline):
-    model = BesluitInformatieObject
-    extra = 0
-    readonly_fields = ("uuid",)
-    raw_id_fields = ("_informatieobject",)
```

Comment:
Now `_informatieobject` is no longer a `raw_id_field` right?

Reply:
`BesluitInformatieObjectInline` has now `EditInlineAdminMixin`, and it seems that now `raw_id_fields` has no effect on the representation of inline object.